README.md

```diff
@@ -52,11 +52,18 @@ cd /path/to/my/front-end/module
 npm publish
 ```
 ![](assets/publish.png)
+
 3. Perform any manual QA in your DEV environment.
 4. Promote the `module@version` to production using `npm dist-tag add`
 ```
 npm dist-tag add module@version prod
 ```
+![](assets/promote.png)
```

Comment:
IMHO we should fill in the alt tag for the images to give a basic description of what it is.